scripts/build-android.js

```diff
@@ -20,6 +20,7 @@ const commandLineArgs = require("command-line-args");
 const fs = require("fs-extra");
 const path = require("path");
 const exec = require("child_process").execFileSync;
+const packageRoot = path.resolve(__dirname, "..");
```

Comment:
Should we move the build scripts into `packages/realm`?

Reply:
Also this will put the build artifacts in `react-native` instead of `packages/realm/react-native`.  I think we want to actually delete the root project `react-native` folder and have everything within `packages/realm/react-native`, right?

Reply:
> move the build scripts into `packages/realm`?

I was thinking about that. I see pros and cons to it. It probably makes sense to increase cohesion as these will only be used to build the SDK native parts. But then again, the CmakeLists.txt live in the bindgen package right now, which sort of serves the same purpose.

One my thought I had was that I want to refactor and join the build scripts into a single CLI that can invoke cmake when building for Node.js, iOS or Android with one (or more) platform and arch flags.